content/applications/websites/ecommerce/managing_products/catalog.rst

```diff
@@ -0,0 +1,184 @@
+an extra level of categorization in your catalog, you can activate various **filters** such as
+attributes or sort-by search.
+
+Attributes
+~~~~~~~~~~
+
+Attributes refer to **characteristics** of a product such as **color** or **material**, whereas
+variants are the different combinations of attributes. :guilabel:`Attributes and Variants` can be
+found under :menuselection:`Website app --> eCommerce --> Products`, select your product, and
+:guilabel:`Attributes & Variants` tab.
+
+.. seealso::
+   - :doc:`../../../sales/sales/products_prices/products/variants`
+
+.. image:: catalog/catalog-attributes.png
+   :align: center
+   :alt: Attributes and variants of your product
+
+To enable **attributess** filtering, go to :menuselection:`Website dashboard --> Shop`, click on
+:menuselection:`Edit --> Customize tab` and select either :guilabel:`Left`, :guilabel:`Top`, or
+both. Additionally, you can also enable :guilabel:`Price Filtering` to enable price filters.
+
+.. note::
+   :guilabel:`Price Filter` works independently from **attributes** and therefore can be enabled on
+   its own if desired.
+
+.. tip::
+   You can use **attribute filters** even if you do not work with product variants. When adding
+   attributes to your products, make sure only to specify *one* value per attribute. Odoo will not
+   create variants if no combination is possible.
+
+Sort-by search
+~~~~~~~~~~~~~~
+
+It is possible to allow the user to manually **sort the catalog** using the search bar. From
+:menuselection:`Website dashboard --> Shop`, click on :menuselection:`Edit --> Customize tab`, you
+can enable or disable the :guilabel:`Sort-By` option as well as the :guilabel:`Layout` button. You
+can also select the :guilabel:`Default Sort` of the :guilabel:`Sort-By` button. The default sort
+applies to *all* categories.
+
+The **sorting** options are:
+
+- Featured
+- Newest Arrivals
+- Name (A-Z)
+- Price - Low to High
+- Price - High to Low
+
+In addition, you can **manually edit** the catalog's order of a products by going to
+:menuselection:`Website dashboard --> Shop page --> Edit` and clicking on the product. Under the
+:guilabel:`Product` section of the :guilabel:`Customize` section, you can rearrange the order by
+clicking on the arrows. `<<` `>>` move the product to **extreme** right or left, and `<` `>` move
+the product by **one** row to the right or left. It is also possible to change the catalog's order
+of products in :menuselection:`Website app --> eCommerce --> Products` and dragging-and-dropping the
+products within the list.
+
+.. image:: catalog/catalog-reorder.png
+   :align: center
+   :alt: Product rearrangement in the catalog
+
+Page design
+===========
+
+Category page
+-------------
+
+You can customize the layout of the category page using the website builder. Note that editing the
+layout of the category page is global; editing one category layout will affect *all* category pages.
```

Comment:
- use admonitions for that kind of comments (you even wrote "note that...")
- slightly rephrased (+ present tense)

```suggestion
You can customize the layout of the category page using the website builder. 

.. important::
   Note that editing the category page layout is global; editing one category layout affects *all* category pages.
```